pkg/resourceinterpreter/configurableinterpreter/luavm/lua.go

```diff
@@ -198,7 +198,7 @@ func (vm VM) Retain(desired *unstructured.Unstructured, observed *unstructured.U
 }
 
 // AggregateStatus returns the objects that based on the 'object' but with status aggregated by lua.
-func (vm VM) AggregateStatus(object *unstructured.Unstructured, item []map[string]interface{}, script string) (*unstructured.Unstructured, error) {
+func (vm VM) AggregateStatus(object *unstructured.Unstructured, item []workv1alpha2.AggregatedStatusItem, script string) (*unstructured.Unstructured, error) {
```

Comment:
```suggestion
func (vm VM) AggregateStatus(object *unstructured.Unstructured, items []workv1alpha2.AggregatedStatusItem, script string) (*unstructured.Unstructured, error) {
```
We should use the plural.